README.md

```diff
@@ -1,409 +1,218 @@
+# Cassandra Migrations
 [![Gem Version](https://badge.fury.io/rb/cassandra_migrations.png)](http://badge.fury.io/rb/cassandra_migrations)
 [![Code Climate](https://codeclimate.com/github/hsgubert/cassandra_migrations.png)](https://codeclimate.com/github/hsgubert/cassandra_migrations)
 
-Cassandra Migrations
-====================
+## Description
+Cassandra Migrations is a Cassandra database schema migration library for Rails applications.
 
-**Cassandra schema management for a multi-environment development.**
+This gem provides:
+  * Multi-environment database configuration
+  * Versioned CQL schema migration management
+  * A schema modification DSL for simplified migration code
+  * Rake tasks for database schema management
+  * Support for forked processes
 
-A gem to manage Cassandra database schema for Rails. This gem offers migrations and environment specific databases out-of-the-box for Rails users.
+Use Cassandra in an organized and familiar way, without changing how you work with your ActiveRecord relational database schema.
 
-This enables you to use Cassandra in an organized way, combined with your ActiveRecord relational database.
+## Requirements
 
-# Requirements
+- Cassandra >= 1.2, using the native transport protocol
+- Ruby >= 1.9
+- Rails >= 3.2
 
-- Cassandra 1.2 or higher with the native_transport_protocol turned on ([Instructions to install cassandra locally](https://github.com/hsgubert/cassandra_migrations/wiki/Preparing-standalone-Cassandra-in-local-machine))
-- Ruby 1.9
-- Rails > 3.2
+## Installation
 
-# Installation
-
-    gem install cassandra_migrations
-
-# Quick start
-
-### Configure Cassandra
-
-The native transport protocol (sometimes called binary protocol, or CQL protocol) is not on by default on all version of Cassandra. If it is not you can enable by editing the `CASSANDRA_DIR/conf/cassandra.yaml` file on all nodes in your cluster and set `start_native_transport` to `true`. You need to restart the nodes for this to have effect.
+```ruby
+gem install cassandra_migrations
+```
 
-### Prepare Project
+or, with bundler, add the following to your gemfile:
 
-In your rails root directory run:
+```
+gem 'cassandra_migrations'
+```
 
-    prepare_for_cassandra .
+## Configuration
 
-Which create the `config/cassandra.yml`
+Similar to how `/config/database.yml` stores your relational databse configuration, `config/cassandra.yml` stores your cassandra database configuration.
 
-### Configuring cassandra access
+```
+rails generate cassandra:configuration
```

Comment:
@hsgubert if you are good with this direction, I'll implement tomorrow. The previous methodology didn't work for me  (file not found)- maybe because I am using bundler. Thought a generator might be better, but defer to you, of course.